ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -0,0 +1,222 @@
+  });
+  const history = useHistory();
+
+  const [dropdownState, setDropdownState] = useState(defaultDropdownState);
+  return (
+    <div className="collectibles-items">
+      <Box padding={[4, 6, 4, 6]} flexDirection={FLEX_DIRECTION.COLUMN}>
+        <>
+          {Object.keys(collections).map((key, index) => {
+            const {
+              collectibles,
+              collectionName,
+              collectionImage,
+            } = collections[key];
+
+            const isExpanded = dropdownState[key];
+            return (
+              <div
+                className="collectibles-items__item"
+                key={`collection-${index}`}
+                onClick={() => {
+                  setDropdownState((_dropdownState) => ({
+                    ..._dropdownState,
+                    [key]: !isExpanded,
+                  }));
+                }}
+              >
+                <Box
+                  marginTop={4}
+                  marginBottom={4}
+                  display={DISPLAY.FLEX}
+                  alignItems={ALIGN_ITEMS.CENTER}
+                  justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+                >
+                  <Box alignItems={ALIGN_ITEMS.CENTER}>
+                    {collectionImage ? (
+                      <img
+                        src={collectionImage}
+                        className="collectibles-items__item__collection-image"
+                      />
+                    ) : (
+                      <div className="collectibles-items__item__collection-image-alt">
+                        {collectionName[0]}
+                      </div>
+                    )}
+                    <Typography
+                      color={COLORS.BLACK}
+                      variant={TYPOGRAPHY.H4}
+                      margin={[0, 0, 0, 2]}
+                    >
+                      {`${collectionName} (${collectibles.length})`}
+                    </Typography>
+                  </Box>
+                  <Box alignItems={ALIGN_ITEMS.FLEX_END}>
+                    <i
+                      className={`fa fa-lg fa-chevron-${
+                        isExpanded ? 'down' : 'right'
+                      }`}
+                    />
+                  </Box>
+                </Box>
+                {isExpanded ? (
+                  <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP}>
+                    {collectibles.map((collectible, i) => {
+                      const {
+                        image,
+                        address,
+                        tokenId,
+                        backgroundColor,
+                      } = collectible;
+                      const collectibleImage = getAssetImageURL(
+                        image,
+                        ipfsGateway,
+                      );
+                      return (
+                        <Box width={width} margin={1} key={`collectible-${i}`}>
+                          <div
```

Comment:
Same thing for this, should this also be an `a` or something similar?

Reply:
added css instead: https://github.com/MetaMask/metamask-extension/pull/12970/commits/3f419b051e8adcf3c77896b9c2ec73e43ce02001

Reply:
This is updated here https://github.com/MetaMask/metamask-extension/pull/12970/commits/5ee5afa3c4f7e1d291638421df0e53ef27df3d12 (still using css to make `cursor: pointer`)